src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/InputLocalFinalVariableName.java

```diff
@@ -188,6 +188,26 @@ void veryLong()
           enough talk */
     }
 
+    // check for try-with-resources
+    void method() throws Exception {
+        final String fileName = "Test";
+        try (BufferedReader bR = new BufferedReader(new InputStreamReader(
```

Comment:
Oh... Now I see your point. I moved all input for try-with-resorces check to separate file. However, I needed to add a one more test, to use this input file. Hope, now it is ok